settings.py

```diff
@@ -114,6 +114,12 @@
     'less#1.7.3',
     'backbone#0.9.1',
     'bootstrap-daterangepicker#2.1.13',
+    'd3#3.1.5',
+    'nvd3#1.1.10-beta',
+    'datatables#1.10.9',
+    'datatables-bootstrap3#0.1',
+    'dangrossman/bootstrap-daterangepicker#7435612692b33b85c72539d37c59bb7edd420c6f',
```

Comment:
@NoahCarnahan @biyeun
```
'bootstrap-daterangepicker#2.1.13',
```
was added above by https://github.com/dimagi/commcare-hq/pull/8727, and these two can't coexist so staging deploy is failing. I'm not sure what to do here so I'll leave it up to you.